<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TraderGUIClass</class>
 <widget class="QMainWindow" name="TraderGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Trader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabMarket">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabLeft">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabRight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGateway">
    <property name="title">
     <string>系统</string>
    </property>
    <widget class="QMenu" name="menuCTP">
     <property name="title">
      <string>CTP</string>
     </property>
     <addaction name="ctpActionLogin"/>
     <addaction name="ctpActionLogout"/>
    </widget>
    <widget class="QMenu" name="menuLST">
     <property name="title">
      <string>LST</string>
     </property>
     <addaction name="actionLogin_2"/>
     <addaction name="actionLogout_2"/>
    </widget>
    <widget class="QMenu" name="menuIB">
     <property name="title">
      <string>IB</string>
     </property>
     <addaction name="actionLogin_3"/>
     <addaction name="actionLogout_3"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuCTP"/>
    <addaction name="menuLST"/>
    <addaction name="menuIB"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuStrategy">
    <property name="title">
     <string>策略管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuRisk">
    <property name="title">
     <string>风险管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menuGateway"/>
   <addaction name="menuStrategy"/>
   <addaction name="menuRisk"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ctpActionLogin">
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="ctpActionLogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionLogin_2">
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionLogout_2">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionLogin_3">
   <property name="text">
    <string>login</string>
   </property>
  </action>
  <action name="actionLogout_3">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TraderGUI.qrc"/>
 </resources>
 <connections/>
</ui>
